extensions/blocks/podcast-player/style.scss

```diff
@@ -445,5 +446,14 @@ $player-background: transparent;
 		.mejs-unmute > button {
 			mask-position: -40px 0;
 		}
+
+		.jetpack-podcast-player__track-status-icon--playing {
+			mask-position: -60px 4px;
```

Comment:
4px y offset to visually vertically center the icon

Reply:
Can we use margin/padding instead?